### ISSUE TITLE
Conda distribuituion

### ISSUE THREAD

**Issue by BrenoJesusFernandes** [pandas-dev/pandas-stubs#49]
@Dr-Irv Could you please tell me, what is the process to publish the package in conda? I finished the CD process to publish in github and pypi automaticly with a new version, but I'm not sure if it is possible to automate the conda process.

**Comment by BrenoJesusFernandes** [comment]
I see, I got it now. Sorry the delay I was at hospital the last two weeks, but I'm ok now. So I'm just gonna merge this PR and start to work in the new issues ;)

By the way with "poetry publish --build" you could do the same work without twine.

**Comment by Dr-Irv** [comment]
> I see, I got it now. Sorry the delay I was at hospital the last two weeks, but I'm ok now. So I'm just gonna merge this PR and start to work in the new issues ;)
> 
Glad to hear you're OK!

> By the way with "poetry publish --build" you could do the same work without twine.

I've published twice. The first time I did `poetry publish`, I got prompted for my username, and it just published the wheel, not the source.  The second time, `poetry publish` prompted me again, but then it hung and never published.  So I switched to doing `poetry build` followed by `twine dist/*` to make it work.

I'm totally new to publishing things on pypi and conda, so help is definitely appreciated!
